indev/libinput.c

```diff
@@ -397,10 +402,8 @@ static void read_pointer(libinput_drv_state_t *state, struct libinput_event *eve
       pointer_event = libinput_event_get_pointer_event(event);
       state->most_recent_touch_point.x += libinput_event_pointer_get_dx(pointer_event);
       state->most_recent_touch_point.y += libinput_event_pointer_get_dy(pointer_event);
-      state->most_recent_touch_point.x = state->most_recent_touch_point.x < 0 ? 0 : state->most_recent_touch_point.x;
-      state->most_recent_touch_point.x = state->most_recent_touch_point.x > LV_HOR_RES - 1 ? LV_HOR_RES - 1 : state->most_recent_touch_point.x;
-      state->most_recent_touch_point.y = state->most_recent_touch_point.y < 0 ? 0 : state->most_recent_touch_point.y;
-      state->most_recent_touch_point.y = state->most_recent_touch_point.y > LV_VER_RES - 1 ? LV_VER_RES - 1 : state->most_recent_touch_point.y;
+      state->most_recent_touch_point.x = LV_MIN(LV_MAX(state->most_recent_touch_point.x, 0), drv->hor_res - 1);
```

Comment:
```suggestion
      state->most_recent_touch_point.x = LV_CLAMP(0, state->most_recent_touch_point.x, drv->hor_res - 1);
```

Reply:
Ohh, I missed the `LV_CLAMP` macro. :facepalm: 

Applied the fix.